index.html

```diff
@@ -6,27 +6,32 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" type="text/css" href="style.css" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;500;700&display=swap">
-    <title>Portfolio page </title>
+    <title>My Portfolio page </title>
 </head>
 <body>
     <div class="main-one">
         <header>
             <h1 class="mylogo">Ndonyi</h1>
-            <menu>
-                <ul class="home">
+            
+                <ul id="home">
                     <li class="content"><a href="#">Portoflio</a></li>
                     <li class="content"><a href="https://citec-47.github.io/frontendmasters/">About</a></li>
                     <li class="content"><a href="https://citec-47.github.io/Desk-version/">Contact</a></li>
                 </ul>
-            </menu>
-          
+              
+                <ul id="home1">
+                    <h1 id="close">X</h1>
+                    <li class="content1"><a href="#">Portoflio</a></li>
+                    <li class="content1"><a href="https://citec-47.github.io/frontendmasters/">About</a></li>
+                    <li class="content1"><a href="https://citec-47.github.io/Desk-version/">Contact</a></li>
+                </ul>
```

Comment:
Good job 👍🏽 on the popup menu but the design is not quite that yet 🤔 you could consider adding some `blur` to the body and making sure that the link colors are `white`, you also have some extra unnecessary margin at the bottom that you should remove ❌ . Take a look at the differences 👇🏽 

## Figma
![Screenshot 2023-03-08 at 00 23 18](https://user-images.githubusercontent.com/45739963/223626593-6b2b2c91-8340-4d27-b08f-eb8078011825.png)

## Your design
![Screenshot 2023-03-08 at 00 23 37](https://user-images.githubusercontent.com/45739963/223626630-f682777d-7675-4847-a88f-517f83fe1dcf.png)